app/src/main/java/org/projectbuendia/client/json/JsonOrdersResponse.java

```diff
@@ -0,0 +1,13 @@
+package org.projectbuendia.client.json;
+
+/**
+ * JSON representation of a set of patients returned by the server.
+ * <p>
+ * TODO: Generify this class into e.g. an IncrementalFetchResponse<JsonEncounter>
+ */
+public class JsonOrdersResponse {
+    public JsonOrder[] results;
+    // TODO(capnfabs): Rename this to syncToken.
+    /** In ISO 8601 date format. */
+    public String snapshotTime;
```

Comment:
Gson supports parsing things into dates (see `org.projectbuendia.client.json.Serializers`), but we shouldn't actually be treating this as a date - that's a legacy thing. It's an opaque string that we're just supposed to send to the server. I'll rename all of these from `snapshotTime` to `syncToken` later this sprint, and then we'll remove the reference to them being dates.